bin/casher

```diff
@@ -141,7 +141,8 @@ function add {
     return
   fi
 
-  paths=$(expand_path "$@")
+  # paths=$(expand_path "$@")
+  paths="$@"
```

Comment:
On closer inspection:

`travis_build` stock logic that uses `~` for cache paths will become broken since Bash won't expand it in `$(checksum_checker)` invocation:

```
$ paths="~ C:\\"
$ md5deep64 ${paths}
C:\Users\Sasha\~: No such file or directory
C:\: Is a directory
```

Reply:
The changes also need to be duplicated into Ruby version since [it's used in preference to Bash version if RVM is available](https://github.com/travis-ci/travis-build/blob/69d86a09ced4080dc79def5bfcd02196105dbc17/lib/travis/build/script/shared/directory_cache/base.rb#L217-L220).